tests/files/acl/test-acl.php

```diff
@@ -44,12 +46,14 @@ public function test__maybe_load_restrictions__no_constant_and_no_options() {
 	public function test__maybe_load_restrictions__no_constant_and_with_one_option() {
 		update_option( 'vip_files_acl_restrict_all_enabled', 1 );
 
-		$this->expectException( \Exception::class );
-		$this->expectExceptionMessage( 'File ACL restrictions are enabled without server configs' );
-
-		maybe_load_restrictions();
-
-		$this->assertEquals( false, has_filter( 'vip_files_acl_file_visibility' ) );
+		try {
+			maybe_load_restrictions();
+			self::assertTrue( false );
```

Comment:
I'm not sure what this is doing exactly.

Reply:
Can we just add a comment there for future reference to why we are doing it this way? Makes sense now, thank you!

Reply:
Also in #4463 too